codelists/views.py

```diff
@@ -73,7 +73,7 @@ def all_valid(self, form, reference_formset, signoff_formset):
             coding_system_id=form.cleaned_data["coding_system_id"],
             description=form.cleaned_data["description"],
             methodology=form.cleaned_data["methodology"],
-            csv_data=form.cleaned_data["csv_data"].read().decode("utf8"),
+            csv_data=form.cleaned_data["csv_data"],
```

Comment:
?

Reply:
In [`CodelistVersionForm.clean_csv_data` (inherited from `CSVValidationMixin`)](https://github.com/opensafely/opencodelists/pull/84/files#diff-dce4bf0e7b1fc6b64bd36f4b7f656403R50) we need to read the contents to validate it.  Since we'd already read the contents into a variable, it seemed natural/in-keeping with how Forms often work to return the contents of that variable.